dlp/src/main/java/com/example/dlp/Inspect.java

```diff
@@ -72,165 +72,6 @@
 
 public class Inspect {
```

Comment:
There's still a lot going on in this snippets class, including some command-line logic. Is this file intended to show the new sample format or just the other files?

Reply:
I only ported over two of the samples into the new format - I'll move the rest over if/when the new format is finalized.